### PR TITLE
Add bionic.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

### CODE REVIEW

File: tests/test_code_format.py [hunk 1]
```diff
@@ -6,9 +6,7 @@ def test_flake8():
     """Test source code for pyFlakes and PEP8 conformance"""
     this_dir = os.path.dirname(os.path.abspath(__file__))
     source_dir = os.path.join(this_dir, '..', 'catkin_tools')
-    cmd = ['flake8', source_dir, '--count', '--max-line-length=120']
-    # work around for https://gitlab.com/pycqa/flake8/issues/179
-    cmd.extend(['--jobs', '1'])
+    cmd = ['flake8', source_dir, '--count', '--max-line-length=120', '--ignore=E722']
```
Comment (dirk-thomas) [inline tests/test_code_format.py]: I would suggest to rather fix the code then ignore this valuable error.

Comment (wjwwood) [reply tests/test_code_format.py]: @clalancette just revert this commit and I'll merge it without green CI. I'll fix up the travis stuff separately.

**Comment by racko** [comment]
Would you mind having a look at other pull requests? :sweat_smile:

File: tests/test_code_format.py (hunk 1 above)
Comment (clalancette) [reply tests/test_code_format.py]: > @clalancette just revert this commit and I'll merge it without green CI. I'll fix up the travis stuff separately.

Sounds good, will do.

**Comment by wjwwood** [comment]
> Would you mind having a look at other pull requests? 😅

Not today. When I have time I will, but I don't know when that will be.

Please don't ask on unrelated issues in the future. If you want to ask, please do so on individual issues and pull requests on which you've been involved.

**Pull request merged**